vdom/apply-properties.js

```diff
@@ -24,6 +26,13 @@ function applyProperties(node, props, previous) {
             }
         }
     }
+
+    // apply hooks after properties
```

Comment:
Is there a test for this?

Reply:
Yes, here it is https://github.com/Matt-Esch/virtual-dom/pull/444/files#diff-524f61060e1d8147246f863be0a1463aR66. May add a couple more tests if you like.

Reply:
It is not obvious to me how deferring hook calls solves the problem.
I was looking for a test that ran a hook and verified that it had been deferred.